vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -100,9 +100,9 @@ function Get-FilesToCacheOnVHD {
             "https://github.com/microsoft/SDN/raw/master/Kubernetes/windows/helper.psm1",
             "https://github.com/Microsoft/SDN/raw/master/Kubernetes/windows/hns.psm1",
             "https://globalcdn.nuget.org/packages/microsoft.applicationinsights.2.11.0.nupkg",
-            "https://acs-mirror.azureedge.net/aks-engine/windows/provisioning/signedscripts-v0.0.2.zip",
```

Comment:
I know 0.0.2 is old enough and not used in even aks-engine, but how do we define out-of-dated? can we remove `v0.0.3` sometime in the future?

Reply:
Good question. I think that we can keep 3 versions or at least 3 months for signed scripts. Let us try it and then refine the principal later.